README.md

```diff
@@ -385,6 +385,7 @@ Parameter | Description
 --dry-run | Run schemachange in dry run mode. The default is 'False'.
 --query-tag | A string to include in the QUERY_TAG that is attached to every SQL statement executed.
 --oauth-config | Define values for the variables to Make Oauth Token requests  (e.g. {"token-provider-url": "https//...", "token-request-payload": {"client_id": "GUID_xyz",...},... })'
+-cfl --change-file-list | Comma delimited list of files, full path, that schemachange will filter to when deploying V and R scripts.
```

Comment:
By "filter", is it filter out? meaning it will not check for the files on this list? Or vice versa? I think you could make this sentence clearer on the flag's purpose